components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -497,11 +516,25 @@ export const PortfolioAsset = (props: Props) => {
     if (nftMetadata && nftDetailsRef?.current) {
       const command: UpdateNFtMetadataMessage = {
         command: NftUiCommand.UpdateNFTMetadata,
-        payload: nftMetadata
+        payload: {
```

Comment:
nit: seems like a lot of these React.useEffect() calls are doing basically the same things with different commands. Could we refactor the commands into a single function and then encapsulate these for better readability?

This isn't security blocking, but seems like a good follow up issue to track later.